README.md

```diff
@@ -319,7 +317,7 @@ Geçici alanlar, sadece belirli koşullar altında değer alırlar. Bu koşullar
 
 Genellikle geçici alanlar, çok fazla girdisi olan bir algoritma içinde kullanılmak için oluşturulurlar. Dolayısıyla, metot için çok fazla parametre geçmek yerine, sınıfın içinde, veriyi tutması için bir alan oluşturulur. Bu alanlar sadece bu algoritma içinde kullanılır ve sonrasında artık anlamsızdır.
 
-Bu tarz bir kodun anlaşılması zordur. Siz sürekli ilgili alanın bir veri tuttuğunu varsayarsınız ama o tek bir algoritma dışında, her zaman boştur.
+Bu tarz bir kodun anlaşılması zordur. Siz sürekli ilgili alanın bir veri tuttuğunu varsayarsınız; ama o tek bir algoritma dışında, her zaman boştur.
```

Comment:
Devam eden cümlede neden : kullandığınızı anlamadım.

Reply:
Birbiriyle ilişkili iki cümleyi ayırmak için kullanılan "ama", "çünkü", "fakat" gibi bağlaçların neredeyse hepsinden önce bir noktalı virgül de kullanmak gerekiyor. Yukarıdaki cümleye dikkat ederseniz (ve hatta sesli okursanız) "ama"'dan hemen önce ufak da olsa bir duraksama yaşadığınızı fark edeceksiniz. İşte o duraksamayı (ve dolayısıyla cümlelerin ayrı olduğunu) belirtmek için kullandım.